CHANGES.rdoc

```diff
@@ -3,6 +3,11 @@
 This document describes the relevant changes between releases of the
 _hawkular-client_ project.
 
+=== V 2.7.0
+
+* Support for SSL connections
+* Fix fetching stats to support rates
+
```

Comment:
Alissa did the release:
https://rubygems.org/gems/hawkular-client/versions/2.7.0
Thanks!